devkit/states/vim/pr_vimrc_init

```diff
@@ -1,22 +1,364 @@
-" Set :GoDef and :GoInfo to use gopls
+" get all of the plugins
+source ~/.pr_vimrc_plug
+
+"-----------------------
+" airblade/vim-gitgutter
+"-----------------------
+" TODO disables default key maps why
```

Comment:
I think it is safe to disable these in preference to using Fugitive

Reply:
done